pkg/distro/fedora/images.go

```diff
@@ -815,6 +816,12 @@ func makeOSTreeParentCommit(options *ostree.ImageOptions, defaultRef string) (*o
 		URL:  options.URL,
 		Ref:  parentRef,
 		RHSM: options.RHSM,
+		MTLS: &ostree.MTLS{
+			CA:         os.Getenv("OSBUILD_SOURCES_OSTREE_SSL_CA_CERT"),
+			ClientCert: os.Getenv("OSBUILD_SOURCES_OSTREE_SSL_CLIENT_CERT"),
+			ClientKey:  os.Getenv("OSBUILD_SOURCES_OSTREE_SSL_CLIENT_KEY"),
+		},
+		Proxy: os.Getenv("OSBUILD_SOURCES_OSTREE_PROXY"),
```

Comment:
This is something I am not sure how to do - for the actual fedora/rhel manifest build, where should I read the configuration for MTLS to do ostree commit resolving? For now I am proposing ENV variables, but I am not sure. From what I know, this library is used in composer and bib, the latter is not relevant so only in composer. Let me know how to properly pass the MTLS configuration.

Reply:
You don't set them at this point, they'll be set by the worker when calling osbuild https://github.com/osbuild/osbuild-composer/blob/6ccfc7f81884eedbf3290bb2118d0c6e31c676d5/cmd/osbuild-worker/jobimpl-osbuild.go#L519-L528.